channel_names: 
     - "DIO4_EF_READ_A"
channel_addresses:
    - 0
    - 2
    - 4
    - 6
    - 8
    - 10
    - 14
config_registers:
    - address: "DIO4_EF_ENABLE"
      value: 0
    - address: "DIO4_EF_INDEX"
      value: 11
    - address: "DIO4_EF_ENABLE"
      value: 1
device_type: "ANY"
conn_type: "ANY"
identifier: "ANY"
use_channel_names: false
streaming: true
verbose: true